process/interceptors/whiteListDataVerifier.go

```diff
@@ -43,6 +45,12 @@ func (w *whiteListDataVerifier) IsWhiteListed(interceptedData process.Intercepte
 // Add adds all the list to the cache
 func (w *whiteListDataVerifier) Add(keys [][]byte) {
 	for _, key := range keys {
+		if len(key) > maxElementSize {
```

Comment:
Do not add elements which are higher than 32

Reply:
you're right, will keep the warn though